File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //Prompt user for appropriate height
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h > 23 || h < 0);

    for (int i = 0; i < h; i++)
    {
        for (int s = 0; s<h-i-1; s++)
        //print spaces h-i-1
        {
            printf(" ");
        }
        //print hashes = i + 2
        for (int x=0; x<i+2;x++)
        {
            printf("#");
        }
        //print new line
        printf("\n");
    }

}

File: pset6/mario/mario.py
from cs50 import get_int

#prompt user for appropraite height
while True:
    h = get_int("Height: ")
    if h >= 0 and h < 24:
        break

#for every row
for i in range(h):
    #print spaces
    print(" " * (h-i-1), end="")
    #print hashes & new line
    print("#" * (i+2))
